src/database/queries/ScheduledItem.ts

```diff
@@ -129,6 +129,14 @@ export async function getItemsForScheduledSurface(
         language: scheduledItem.approvedItem.language,
         publisher: scheduledItem.approvedItem.publisher,
         imageUrl: scheduledItem.approvedItem.imageUrl,
+        // so the type definition in /src/database/types has topic as optional,
+        // which typescript resolves as `string | undefined`. however, if the
+        // topic is missing in the db, prisma returns `null` - hence the
+        // nullish coalescing operator below.
+        //
+        // i wonder why typescript won't accept both. is there some deep dark
+        // JS reason? or is it just better practice?
```

Comment:
+1 vote for the "deep dark JS reason" here! :) We could set the type in `/src/database/types` to `topic: string | null | undefined`, but I'm not sure if it would create more problems than it solved!